src/bin/rav1e.rs

```diff
@@ -137,6 +137,10 @@ fn main() {
     video_info.time_base.num as usize
   );
 
+  if cli.limit != 0 {
+    cli.limit -= cli.skip ;
```

Comment:
If skip is larger than limit rust would rightfully panic :) You have a check below you should use here.

Reply:
When we 

> @vibhoothiiaanand Thanks for implementing this!
> 
> Personally, I would like to keep `--limit` always the number of frames to encode, so that `--skip 10 --limit 20` encodes frames 11 to 30. This is in line with most other encoders and decoders.

Thanks for the suggestions, I did make the changes and I guess it is now working as you said.

```
cargo run --release --bin rav1e  /Volumes/buildBox/rav1e/dummyData/ducks_take_off_420_720p50.y4m --skip 10 --limit 5  -o test.ivf -v
   Compiling rav1e v0.1.0 (/Volumes/buildBox/rav1e/rav1e)
    Finished release [optimized] target(s) in 52.98s
     Running `target/release/rav1e /Volumes/buildBox/rav1e/dummyData/ducks_take_off_420_720p50.y4m --skip 10 --limit 5 -o test.ivf -v`
1280x720 @ 50/1 fps
Frame 0 - Key frame - 143179 bytes - encoded 1/5 frames, 0.102 fps, 55929.30 Kb/s, est. size: 0.68 MB, est. time: 39 s
Frame 1 - Inter frame - 135280 bytes - encoded 2/5 frames, 0.056 fps, 54386.52 Kb/s, est. size: 0.66 MB, est. time: 54 s
Frame 2 - Inter frame - 5 bytes - encoded 3/5 frames, 0.084 fps, 36258.33 Kb/s, est. size: 0.44 MB, est. time: 24 s
Frame 3 - Inter frame - 2123 bytes - encoded 4/5 frames, 0.094 fps, 27401.07 Kb/s, est. size: 0.33 MB, est. time: 11 s
Frame 4 - Inter frame - 5 bytes - encoded 5/5 frames, 0.118 fps, 21921.25 Kb/s, est. size: 0.27 MB, est. time: 0 s

Key Frames:      1    avg size:  143179 B
Inter:           4    avg size:   34353 B
Intra Only:      0    avg size:       0 B
Switch:          0    avg size:       0 B
```